cw2dmk.c

```diff
@@ -1759,6 +1759,40 @@ parse_tracks(const char *nptr, int *rav)
   return err;
 }
 
+/*
+ * Like strtol, but exit with a fatal error message if there are any
+ * invalid characters or the string is empty.
+ */
+long int
+strtol_strict(const char *nptr, int base, const char name)
```

Comment:
The `const` on `const char name` doesn't do anything for the function's API since `name` isn't a pointer.  It'd be the same as doing `const int base`.  But it doesn't hurt anything either.  Up to you if you leave it as is.

However, I'd recommend changing it to a `const char *name` (and adjusting the `fatal_msg` call).  We might eventually want to support long form command line arguments (e.g. `--verbosity`), so using a string arg could help support that eventuality.

Reply:
Changing it to take a string would make it harder to use, as the call sites only have a char. Hmm, well, I could create a string just above the switch (ch). That's not so bad.

(The unneeded const is left from my first draft code where the name argument actually was a const char*. I changed that almost immediately when I noticed the callers only had a char, but missed removing const.)

Reply:
I've taken this suggestion in 2c1018b